extensions/cornerstone-dicom-sr/src/viewports/OHIFCornerstoneSRViewport.tsx

```diff
@@ -537,71 +508,41 @@ function _getStatusComponent({
       );
       break;
     case 3:
-      StatusIcon = () => (
-        <div
-          className="flex items-center justify-center -mr-1 bg-white rounded-full group-hover:bg-customblue-200"
-          style={{
-            width: '18px',
-            height: '18px',
-            border: 'solid 1.5px #000000',
-          }}
-        >
-          <Icon
-            name="arrow-left"
-            style={{ color: '#000', width: '14px', height: '14px' }}
-          />
-        </div>
-      );
+      StatusIcon = () => <Icon name="status-untracked" />;
 
-      ToolTipMessage = () => <div>Click to restore measurements.</div>;
+      ToolTipMessage = () => (
+        <div>{`Click ${loadStr} to restore measurements.`}</div>
+      );
   }
 
-  const StatusPill = () => (
-    <div
-      className={classNames(
-        'group relative flex items-center justify-center px-2 rounded-full cursor-default bg-customgreen-100',
-        {
-          'hover:bg-customblue-100': state === 3,
-          'cursor-pointer': state === 3,
-        }
+  const StatusArea = () => (
+    <div className="flex h-6 leading-6 cursor-default text-sm text-white">
+      <div className="min-w-[45px] flex items-center p-1 rounded-l-xl rounded-r bg-customgray-100">
+        <StatusIcon />
+        <span className="ml-1">SR</span>
+      </div>
+      {state === 3 && (
+        <div
+          className="ml-1 px-1.5 rounded cursor-pointer hover:text-black bg-primary-main hover:bg-primary-light"
+          // Using onMouseUp here because onClick is not working when the viewport is not active and is styled with pointer-events:none
```

Comment:
This was weird... there are a few other `onClick` listeners in the `ViewportActionBar` that work fine. I have no idea what is different about this.